imposter.go

```diff
@@ -29,32 +31,53 @@ func parseHybridAddress(s string) (ip net.IP, err error) {
 type HTTPRequest struct {
 	// RequestFrom is the originating address of the incoming request.
 	RequestFrom net.IP
+
 	// Method is the HTTP request method.
 	Method string
+
 	// Path is the path of the request, without the query parameters.
 	Path string
+
 	// Query contains the URL query parameters of the request.
-	// Note that more than one value per key is not supported.
-	Query map[string]string
+	Query url.Values
+
 	// Headers contains the HTTP headers of the request.
-	// Note that more than one value per key is not supported.
-	Headers map[string]string
+	Headers http.Header
+
 	// Body is the body of the request.
 	Body interface{}
+
 	// Timestamp is the timestamp of the request.
 	Timestamp string
 }
 
 // httpRequestDTO is a data transfer object used as an intermediary value
 // for marshalling and un-marshalling the JSON structure of an HTTPRequest.
 type httpRequestDTO struct {
-	RequestFrom string            `json:"requestFrom,omitempty"`
-	Method      string            `json:"method,omitempty"`
-	Path        string            `json:"path,omitempty"`
-	Query       map[string]string `json:"query,omitempty"`
-	Headers     map[string]string `json:"headers,omitempty"`
-	Body        interface{}       `json:"body,omitempty"`
-	Timestamp   string            `json:"timestamp,omitempty"`
+	RequestFrom string                 `json:"requestFrom,omitempty"`
+	Method      string                 `json:"method,omitempty"`
+	Path        string                 `json:"path,omitempty"`
+	Query       map[string]interface{} `json:"query,omitempty"`
+	Headers     map[string]interface{} `json:"headers,omitempty"`
+	Body        interface{}            `json:"body,omitempty"`
+	Timestamp   string                 `json:"timestamp,omitempty"`
+}
+
+// toMapValues maps an HTTP query or header value to its moutebank JSON representation.
```

Comment:
typo: moutebank -> mountebank

Reply:
Fixed the typo - https://github.com/senseyeio/mbgo/pull/11/commits/ab886ca44857f53c76e921341a9dc080064a7fb3.